full-stack-tests/presentation/src/Utils.ts

```diff
@@ -111,6 +111,57 @@ export function insertDocumentPartition(db: IModelDb, code: string, label?: stri
   return { className: "BisCore:DocumentPartition", id };
 }
 
+/** Insert a model element into created imodel. Return created element's className and Id. */
+export function insertPhysicalModel(db: IModelDb, label: string, parentId?: Id64String) {
```

Comment:
Let's move these iModel setup functions into a separate source file, e.g. `IModelSetupUtils.ts` or something like that.

Reply:
moved